### DISCUSSION TITLE
p/8956b229025844859429e252f9591080/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#40] [Announcements]
# p/8956b229025844859429e252f9591080/

场景吾辈最近从 docusaurus 迁移到了更好更快的 vitepress，由于也需要生成 rss，但又找不到合适的插件，所以参考网络教程写了一下。虽然之前使用 docusaurus 时也是自行实现，但使用 vitepress 的 build hooks 还是踩到了新的坑。 实现参考 网络上的文章 和 vue blog，主要思路是在构建时能够拿到所有 html，然后使用 feed 生成需要的 r

https://blog.rxliuli.com/p/8956b229025844859429e252f9591080/

**Comment by xupt-chenxi** [comment]
吾辈，希望能交换友链<br>
昵称：chenxi<br>
站点URL：https://chenxi-space.com<br>
站点名称：chenxi-space<br>
主人头像：https://pic.imgdb.cn/item/65919713c458853aef5343e9.jpg<br>
站点描述：行到水穷处，坐看云起时

**Reply by rxliuli** [reply]
ok，稍等，吾辈改下配置。

**Reply by rxliuli** [reply]
已加
https://blog.rxliuli.com/links/

<img width="957" alt="image" src="https://github.com/rxliuli/blog/assets/24560368/d914a71f-aa72-4c1d-a77f-4f7b09e09737">

**Reply by xupt-chenxi** [reply]
好好好<br>我也将吾辈添加到友链了<br>https://chenxi-space.com/docs/friend.html
